### ISSUE TITLE
Add Algolia Docsearch

### ISSUE THREAD

**Issue by mlrawlings** [mlrawlings/markojs-website#1]
https://community.algolia.com/docsearch/

**Comment by newyork-anthonyng** [comment]
Any progress on this?

I was searching for `subscribe` on the `markojs` website and was met with 0 results. I had to search for `subscribeTo` 😢

I think the owner of the website needs to sign up/register with `algolia`? But I'd be more than happy to help integrate it.

**Comment by austinkelleher** [comment]
I agree this is something that we need to add. I'll look into it soon.
